internal/learn/about-iota/coordinator.md

```diff
@@ -0,0 +1,23 @@
+---
+id: coordinator
+title: Coordinator
+description: What is Coordinator in the IOTA network.
+---
+
+# Coordinator
+
+:::note
+
+Coordinator is a temporary solution. IOTA will remove Coordinator in the future [IOTA 2.0 update](./roadmap-to-decentralization.md).
+
+:::
+
+Coordinator is a client that sends signed messages called milestones that nodes trust and use to confirm messages. Messages in the Tangle are considered for confirmation only when they are directly or indirectly referenced by a milestone that nodes have validated.
```

Comment:
```suggestion
The Coordinator is a client that sends signed messages called milestones that nodes trust and use to confirm messages. Messages in the Tangle are only considered for confirmation when they are directly or indirectly referenced by a milestone that nodes have validated.
```

Reply:
this could use a link to What is Chrysalis , https://wiki.iota.org/chrysalis-docs/introduction/what_is_chrysalis#new-milestone-selection-algorithm

Reply:
I don't like it. Crysalis is the reality for the long time now and the article still talks about it as of a new feature. I believe that not a single newcomer cares about the fact the current revision of IOTA is called Crysalis or how awesome it used to be compared to IOTA 1.0.